src/map.ts

```diff
@@ -0,0 +1,69 @@
+import 'leaflet/dist/leaflet.css';
+import L, {Map} from "leaflet";
```

Comment:
Don't bother with importing `{Map}` unless you're going to do away with `L` completely